.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,40 @@
+name: Feature Request
+description: Suggestions for new or different behavior.
```

Comment:
i noticed that @gratcliff was creating tickets with this title, should we templatize this for features and bugs?

```suggestion
description: Suggestions for new or different behavior.
title: 'Feature Request: '
```

Reply:
i'm cool with that change! honestly it feels a bit moot since we're automatically attaching labels and it's probably better to filter by those anyways

Reply:
"Feature Request:" does add a ton of noise to bug ticket titles. what about `feat:` and `bug:` instead?

Reply:
yes big fan of that change